### PR TITLE
BUG: Consolidate extension metadata with one set in extension build-system

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

**Comment by jcfr** [comment]
For reference, here is the list of pull requests updating metadata in the corresponding extension build-system:

* https://github.com/naterex23/SlicerAblationPlanner/pull/3
* https://github.com/jamesobutler/ModelClip/pull/2
* https://github.com/lassoan/AnomalousFiltersExtension/pull/2
* https://github.com/SlicerIGT/PercutaneousApproachAnalysis/pull/1
* https://github.com/QIICR/PkModeling/pull/11
* https://github.com/SlicerIGT/PortPlacement/pull/3
* https://github.com/netstim/SlicerANTs/pull/9
* https://github.com/jcfr/SlicerToKiwiExporter/pull/9
* https://github.com/SlicerDMRI/SlicerWMA-OLD/pull/12
* https://github.com/NrgXnat/XNATSlicer/pull/3
* https://github.com/Slicer/SlicerAirwaySegmentation/pull/1
* https://github.com/DCBIA-OrthoLab/AnglePlanes-Extension/pull/21
* https://github.com/pzaffino/SlicerArduinoController/pull/3
* https://github.com/Atracsys/SlicerAstmPhantomTest/pull/4
* https://github.com/ToothAndClaw/SlicerAuto3dgm/pull/7
* https://github.com/Auditory-Biophysics-Lab/Slicer-BoneThicknessMapping/pull/7
* https://github.com/jamesobutler/SlicerBrainVolumeRefinement/pull/1
* https://github.com/lassoan/SlicerBreastImplantAnalyzer/pull/1
* https://github.com/ikolesov/CarreraSlice/pull/13
* https://github.com/fedorov/ChangeTrackerPy/pull/20
* https://github.com/acil-bwh/SlicerCIP/pull/43
* https://github.com/lassoan/SlicerCleverSegmentation/pull/1
* https://github.com/DCBIA-OrthoLab/CMFreg/pull/39
* https://github.com/tokjun/CurveMaker/pull/17
* https://github.com/DCBIA-OrthoLab/DatabaseInteractorExtension/pull/12
* https://github.com/SlicerRt/SlicerDebuggingTools/pull/17
* https://github.com/gaudot/SlicerDentalSegmentator/pull/2
* https://github.com/Slicer/SlicerDeveloperToolsForExtensions/pull/30
* https://github.com/pnlbwh/SlicerDiffusionQC/pull/49
* https://github.com/lassoan/SlicerDRRGenerator/pull/2
* https://github.com/QIICR/DSC_Analysis/pull/5
* https://github.com/DCBIA-OrthoLab/EasyClip-Extension/pull/14
* https://github.com/tokjun/ErodeDilateLabel/pull/3
* https://github.com/SlicerRt/FilmDosimetryAnalysis/pull/3
* https://github.com/SlicerRt/GelDosimetryAnalysis/pull/11
* https://github.com/FredericBr/SlicerGeodesic/pull/4
* https://github.com/lpfang/GyroGuide/pull/2
* https://github.com/ImagingDataCommons/SlicerIDCBrowser/pull/35
* https://github.com/gpernelle/iGyne/pull/4
* https://github.com/finetjul/ImageMaker/pull/2
* https://github.com/PerkLab/SlicerMatlabBridge/pull/8
* https://github.com/SlicerMorph/SlicerMEMOS/pull/19
* https://github.com/DCBIA-OrthoLab/MeshStatisticsExtension/pull/18
* https://github.com/NIRALUser/MeshToLabelMap/pull/12
* https://github.com/MHubAI/SlicerMRunner/pull/4
* https://github.com/NIRALUser/3DMetricTools/pull/14
* https://github.com/Project-MONAI/MONAILabel/pull/1675
* https://github.com/Project-MONAI/SlicerMONAIViz/pull/38
* https://github.com/SlicerProstate/mpReview/pull/180
* https://github.com/kyliekeijzer/Slicer-PET-MUST-segmenter/pull/4
* https://github.com/needlefinder/NeedleFinder/pull/2
* https://github.com/NVIDIA/ai-assisted-annotation-client/pull/109
* https://github.com/OrthodonticAnalysis/SlicerOrthodonticAnalysis/pull/4
* https://github.com/aangelos28/PBNRR/pull/6
* https://github.com/PerkTutor/PerkTutor/pull/76
* https://github.com/QIICR/Slicer-PETDICOMExtension/pull/23
* https://github.com/QIICR/PET-IndiC/pull/7
* https://github.com/gti-fing/SlicerPetSpectAnalysis/pull/5
* https://github.com/QIICR/PETTumorSegmentation/pull/22
* https://github.com/DCBIA-OrthoLab/PickAndPaintExtension/pull/25
* https://github.com/KitwareMedical/SlicerPipelines/pull/25
* https://github.com/fepegar/SlicerPyTorch/pull/16
* https://github.com/DCBIA-OrthoLab/Q3DCExtension/pull/106
* https://github.com/QIICR/QuantitativeReporting/pull/275
* https://github.com/gsi-biomotion/SlicerRegistrationQA/pull/17
* https://github.com/R-Vessel-X/SlicerRVXVesselnessFilters/pull/7
* https://github.com/Sunderlandkyl/ScatteredTransform/pull/1
* https://github.com/gaoyi/SegmentationAidedRegistration/pull/8
* https://github.com/mazurowski-lab/SlicerSegmentWithSAM/pull/16
* https://github.com/NIRALUser/ShapePopulationViewer/pull/64
* https://github.com/DCBIA-OrthoLab/ShapeVariationAnalyzer/pull/70
* https://github.com/KitwareMedical/SlicerSkeletalRepresentation/pull/63
* https://github.com/PerkLab/SlicerSkinMouldGenerator/pull/3
* https://github.com/hina-shah/SlicerBatchAnonymize/pull/21
* https://github.com/JoostJM/SlicerCaseIterator/pull/21
* https://github.com/KitwareMedical/SlicerCBCTToothSegmentation/pull/4
* https://github.com/MedicalImageAnalysisTutorials/SlicerCervicalSpine/pull/8
* https://github.com/MedicalImageAnalysisTutorials/SlicerCochlea/pull/43
* https://github.com/DCBIA-OrthoLab/SlicerConda/pull/5
* https://github.com/SlicerDMRI/SlicerDcm2nii/pull/10
* https://github.com/DCBIA-OrthoLab/SlicerDentalModelSeg/pull/30
* https://github.com/QIICR/SlicerDevelopmentToolbox/pull/45
* https://github.com/SlicerDMRI/SlicerDMRI/pull/235
* https://github.com/SlicerFab/SlicerFab/pull/15
* https://github.com/PerkLab/SlicerFreeSurfer/pull/20
* https://github.com/SlicerIGT/SlicerIGT/pull/219
* https://github.com/QIICR/SlicerLayoutButtons/pull/3
* https://github.com/ALive-research/Slicer-Liver/pull/265
* https://github.com/KitwareMedical/SlicerLookingGlass/pull/27
* https://github.com/netstim/SlicerNetstim/pull/18
* https://github.com/HOA-2/SlicerNeuroSegmentation/pull/25
* https://github.com/PerkLab/SlicerOpenAnatomy/pull/20
* https://github.com/Slicer/SlicerOpenCV/pull/79
* https://github.com/jamesobutler/SlicerRegularizedFastMarching/pull/1
* https://github.com/SlicerRt/SlicerRT/pull/243
* https://github.com/sjh26/SlicerThemes/pull/3
* https://github.com/MarinaSandonis/SlicerTissueSegmentation/pull/2
* https://github.com/KitwareMedical/SlicerVirtualMouseCursor/pull/3
* https://github.com/KitwareMedical/SlicerVirtualReality/pull/174
* https://github.com/vmtk/SlicerExtension-VMTK/pull/114
* https://github.com/NIRALUser/SPHARM-PDM/pull/86
* https://github.com/JBeninca/SlicerStereotaxia/pull/10
* https://github.com/mturja-vf-ic-bd/SlicerSurfaceLearner/pull/9
* https://github.com/SlicerHeart/SlicerSurfaceMarkup/pull/14
* https://github.com/gattia/Slicer-T2mapping/pull/5
* https://github.com/QIICR/TCIABrowser/pull/69
* https://github.com/SlicerMicro/Slicer-TITAN/pull/4
* https://github.com/faustomilletari/TOMAAT-Slicer/pull/16
* https://github.com/fsemerar/SlicerTomoSAM/pull/18
* https://github.com/fepegar/SlicerTorchIO/pull/15
* https://github.com/pnlbwh/ukftractography/pull/159
* https://github.com/PerkLab/SlicerVolumeClip/pull/3
* https://github.com/SlicerIGT/ResectionPlanner/pull/5
* https://github.com/SlicerRt/SegmentRegistration/pull/9
* https://github.com/moselhy/SlicerSequenceRegistration/pull/6
* https://github.com/sebastianandress/Slicer-SurfaceFragmentsRegistration/pull/5
* https://github.com/sebastianandress/Slicer-SurfaceWrapSolidify/pull/9
* https://github.com/QIICR/T1Mapping/pull/2
* https://github.com/Slicer/SkullStripper/pull/12
* https://github.com/SlicerIGT/aigt/pull/58
* https://github.com/lassoan/SlicerAutoscroll/pull/1
* https://github.com/openigtlink/SlicerOpenIGTLink/pull/131
* https://github.com/ETS-vis-interactive/SlicerPRISMRendering/pull/35
* https://github.com/SlicerProstate/SlicerProstate/pull/56
* https://github.com/KitwareMedical/SlicerPythonTestRunner/pull/7
* https://github.com/AIM-Harvard/SlicerRadiomics/pull/81
* https://github.com/harellab/SlicerUHFMRTools/pull/10
* https://github.com/VASST/VASSTAlgorithms/pull/4

File: SlicerWMA.s4ext [hunk 1]
```diff
@@ -6,9 +6,9 @@ depends NA
 
 build_subdirectory inner-build
 
-homepage https://github.com/SlicerDMRI/whitematteranalysis
+homepage https://github.com/SlicerDMRI/SlicerWMA#readme
```
Comment (jamesobutler) [inline SlicerWMA.s4ext:9]: It appears the SlicerWMA repo was recently named to “SlicerWMA-OLD”.
https://github.com/SlicerDMRI/SlicerWMA-OLD

@jhlegarreta Is this an indication that SlicerDMRI wants to remove the SlicerWMA extension from the Slicer extensions index? Or will there be a new repo for SlicerWMA?

Comment (jhlegarreta) [reply SlicerWMA.s4ext:9]: Apologies James, but I was/am not aware of the potential related conversations: @jcfr is probably in a better position to answer the question.

Comment (jcfr) [reply SlicerWMA.s4ext:9]: @jhlegarreta Do you happen to know who renamed the repository from `SlicerWMA` to `SlicerWMA-OLD`? The issue I created is intended to help understand the rational and provide some clarification.

**Comment by jcfr** [comment]
In the meantime, I will move forward with the integration.

File: SlicerWMA.s4ext (hunk 1 above)
Comment (jhlegarreta) [reply SlicerWMA.s4ext:9]: Wow, I thought it was you JC. Really sorry for having assumed that. It was not me, and I had no discussion about renaming it with anybody: I only realized about the renaming after I got notified by GitHub about the issue you opened, and thus I mistakenly linked both things.

Comment (pieper) [reply SlicerWMA.s4ext:9]: @ljod and I have been working with @zhangfanmark to update this.  So this extension (SlicerWMA) is being deprecated in favor of https://github.com/SlicerDMRI/SlicerTractParcellation which is still being worked on https://github.com/Slicer/ExtensionsIndex/pull/2037

**Pull request merged**